src/server.ts

```diff
@@ -701,7 +707,13 @@ export class Server {
    * requires a memo, the promise will throw {@link AccountRequiresMemoError}.
    * @throws  {AccountRequiresMemoError}
    */
-  public async checkMemoRequired(transaction: Transaction): Promise<void> {
+  public async checkMemoRequired(
```

Comment:
Hi @abuiles, if the operation destination is an M.. address, can Horizon load it correctly? Or should we skip this single check?

https://github.com/stellar/js-stellar-sdk/pull/535/files#diff-916156c481141bfcd08c6ae40ce10e00R742

Reply:
@overcat that's a great question, I believe we should skip the check if the destination is an `M` address since `M` addresses essentially encode a memo within them

Reply:
@overcat good catch!